seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -47,14 +47,18 @@ arbitrum,zigzag,1,2022-10-26,0xc36027e8ccac981439cae6471824c08fad2831a94d1824505
+ethereum,DODO,0,2022-12-17,0xb7f4dda03cacea98dc5a344caad9d05003bfdf2dbe24eb87123123348dea9574,104,0x70d2b7c19352bb76e4409858ff5746e500f2b67c,401888.80312752,0xc55913df7471e01f2849003b718f7aac1747a107,0.8740885436420626
+ethereum,DODO,0,2022-11-01,0xd82b748ae00631bbb7ffd82e674ff912ae0bc27b6fdd2363a1f0db853761af32,11,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,1000,0xfca59cd816ab1ead66534d82bc21e7515ce441cf,196.3796531780098
 ethereum,DODO,1,2020-09-01,0x784afe0cd29926a86e15390c6d894eccfd3fcfd3d170cbb69a1a9aeb9f98e5af,72,0xc00e94cb662c3520282e6f5717214004a7f26888,24.88,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,6557.285496
 ethereum,DODO,1,2021-01-30,0x616b4f09c34a3e598b00373a961e1daa4c2decc7a5315192f2927a68e767a4d6,100,0xc00e94cb662c3520282e6f5717214004a7f26888,22.84,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,7017.918
-ethereum,DODO,1,2021-01-04,0xffd4a6210d03ff300d64353ded3c2893a8229ed936bbb45bb900fa5829c400d9,108,0xdac17f958d2ee523a2206206994597c13d831ec7,1200,0x43dfc4159d86f3a37a5a4b3d4580b888ad7d4ddd,6023.304990110809067299
-ethereum,DODO,1,2021-03-03,0x4b0a2f323407e746e79521b5e00e3d532b2b99e494078c2804aa77a4943dcaa1,47,0xdac17f958d2ee523a2206206994597c13d831ec7,50000,0x43dfc4159d86f3a37a5a4b3d4580b888ad7d4ddd,11889.181582862616936536
-ethereum,DODO,2,2022-07-24,0x3d01d3959d15fef7be9876f578f87c0c654e2efafb2d551421127e593e0f6b8c,673,0x0391d2021f89dc339f60fff84546ea23e337750f,186,0x990f341946a3fdb507ae7e52d17851b87168017c,500.135950930575867903
-ethereum,DODO,2,2021-06-03,0x399c234917228e17b4f730a4cd5522d198a7e5f868ba147a8e63676e7d41d17e,66,0x437daacccf7bfafce1ac5828804e2c792242cd16,68.63418,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,33.964561
-ethereum,DODO,2,2021-08-26,0xd534643d2f49ee7c7d0020a4e17e9b9e8d7f921e110779fa5c5bd1865e27cb83,290,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,100000,0x6b175474e89094c44da98b954eedeac495271d0f,99930.038478836639848083
-ethereum,DODO,2,2021-05-14,0xb386cb8c8412be350f24c05360e29f3319a903363e4d34076352bb35b44bc758,62,0xdac17f958d2ee523a2206206994597c13d831ec7,75000,0x57ab1ec28d129707052df4df418d58a2d46d5f51,74754.450749991178425833
+ethereum,DODO,2_dvm,2022-11-01,0xd22f88ac18cec7940d5d238140e8cd70e1fb77f01524e92cdba9002831405e53,48,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,823.892847,0x4bdcb66b968060d9390c1d12bd29734496205581,15050.127244362413
+ethereum,DODO,2_dsp,2022-11-01,0x217a50ce8c056b6f240ca0fee82a7aea76fde5845238717d4d2fe36a1d885fe3,18,0x6b175474e89094c44da98b954eedeac495271d0f,16786.79842759679,0xdac17f958d2ee523a2206206994597c13d831ec7,16789.191028
+bnb,DODO,0,2022-10-27,0x14b34175b6f62736909e0ce38149522f17e1549a9b1502c439921f69ce51236b,129,0x26734add0650719ea29087fe5cc0aab81b4f237d,21053,0x55d398326f99059ff775485246999027b3197955,473.56624201412774
+bnb,DODO,0,2022-10-27,0x89d627300b894d58f00467077f9d040465cf2b3081dbe6336f693e9bf1e358e8,195,0x55d398326f99059ff775485246999027b3197955,498,0x30f1183dec479387c09eab6dab3db88cffad5366,1630.1086595819775
```

Comment:
We may have unmatched seed records here as shown in [dbt-test info](https://github.com/duneanalytics/spellbook/actions/runs/3723044188/jobs/6314313730#step:12:148).

Reply:
update dodo_bnb_schema.yml. version=0 should be contained in dodo_aggregator_bnb_trades

Reply:
You may need to modify the `version` in the seed file as well.